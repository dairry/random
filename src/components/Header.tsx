import React, { memo } from 'react';

function Header() {
  return (
    <h1>
      Random{' '}
      <sup
        className="warning tooltip"
        data-tooltip="Generator of random positive integers in a given range"
      >
        *
      </sup>
    </h1>
  );
}

export default memo(Header);
