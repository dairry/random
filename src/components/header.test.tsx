import React from 'react';
import { render } from '@testing-library/react';
import Header from '~components/Header';

describe('Header component', () => {
  it('Header should take a snapshot', () => {
    const { asFragment } = render(<Header />);

    expect(asFragment()).toMatchSnapshot('Header');
  });
});
