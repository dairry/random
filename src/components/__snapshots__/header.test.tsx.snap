// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component Header should take a snapshot: Header 1`] = `
<DocumentFragment>
  <h1>
    Random number generator 
    <sup
      class="warning tooltip"
      data-tooltip="Generator of random positive integers in a given range"
    >
      *
    </sup>
  </h1>
</DocumentFragment>
`;
